src/iOS/Avalonia.iOS/PlatformSettings.cs

```diff
@@ -23,7 +23,7 @@ public override PlatformColorValues GetColorValues()
             ColorContrastPreference.High :
             ColorContrastPreference.NoPreference;
 
-        UIColor? tintColor = null;
+        UIColor tintColor = null;
```

Comment:
I think nullable can be easily enabled in this file.
Either way, let's not remove "?" from code, better to silence this specific warning or even better complete nullability in the file.